neonvm/tools/vm-builder/files/Dockerfile.img

```diff
@@ -65,6 +65,12 @@ RUN set -e \
 		openssh-server \
 	&& /helper.move-bins.sh sshd ssh-keygen
 
+# quota tools
+RUN set -e \
+	&& apk add --no-cache --no-progress --quiet \
+		quota-tools \
+	&& /helper.move-bins.sh quota edquota quotacheck quotaoff quotaon quotastats setquota repquota tune2fs
```

Comment:
`tune2fs` – needed to tune ext4 fs to enable quotas
`setquota` – needed to actually set project quota
`repquota` – can be used to look at current quota usage

Other binaries seem useful, but I've actually not used them and don't know if they're necessary. In theory they can be removed, but may be useful in the future.

What do you think? Should we remove irrelevant binaries from `/helper.move-bins.sh` here? Or should we leave it as is for now?